datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -302,7 +316,8 @@ public void start(final @NonNull DataCapturingListener listener, final @NonNull
      * @throws CursorIsNullException If {@link ContentProvider} was inaccessible.
      */
     @SuppressWarnings("WeakerAccess") // This life-cycle method is called by sdk implementing apps (e.g. SR)
-    public void stop(final @NonNull ShutDownFinishedHandler finishedHandler)
+    public void stop(final @NonNull DataCapturingListener listener,
```

Comment:
before we only registered the CapturingListeners in start() but not in resume() and did not unregister them in pause() and stop() … or did I miss-understand how something ??

Reply:
Wasn't this moved to the constructor?

Reply:
=> Implemented